### PR TITLE
CollectionWidget: replace pointerdown event with pointerup

### PR DESCRIPTION
**Pull request by marker-dao**

### CODE REVIEW

**Review by ksercs — DISMISSED** [review]

**Review by Krijovnick — DISMISSED** [review]

**Review by Krijovnick — DISMISSED** [review]

File: js/ui/collection/ui.collection_widget.base.js [hunk 1]
```diff
@@ -694,19 +694,19 @@ const CollectionWidget = Widget.inherit({
     _attachClickEvent: function() {
         const itemSelector = this._itemSelector();
         const clickEventNamespace = addNamespace(clickEventName, this.NAME);
-        const pointerDownEventNamespace = addNamespace(pointerEvents.down, this.NAME);
+        const pointerEventNamespace = addNamespace(pointerEvents.up, this.NAME);
```
Comment (ksercs) [inline js/ui/collection/ui.collection_widget.base.js]:
```suggestion
        const pointerUpEventNamespace = addNamespace(pointerEvents.up, this.NAME);
```

File: js/ui/collection/ui.collection_widget.base.js [hunk 2]
```diff
@@ -694,19 +694,19 @@ const CollectionWidget = Widget.inherit({
...
         const that = this;
 
-        const pointerDownAction = new Action(function(args) {
+        const pointerAction = new Action(function(args) {
```
Comment (ksercs) [inline js/ui/collection/ui.collection_widget.base.js]:
```suggestion
        const pointerUpAction = new Action(function(args) {
```

**Review by ksercs — DISMISSED** [review]

File: js/ui/collection/ui.collection_widget.base.js (hunk 1 above)
Comment (marker-dao) [reply js/ui/collection/ui.collection_widget.base.js]: [20c312de543201c4ec87ccce694ef2ed96555cea].

File: js/ui/collection/ui.collection_widget.base.js (hunk 2 above)
Comment (marker-dao) [reply js/ui/collection/ui.collection_widget.base.js]: [20c312de543201c4ec87ccce694ef2ed96555cea].

**Review by ksercs — APPROVED** [review]

**Review by Krijovnick — APPROVED** [review]

**Pull request merged**